src/tests/integration/test_generic_db_opeations.py

```diff
@@ -1,28 +1,39 @@
 import pytest
-# flake8: noqa
-from fixtures.database import *
 
 
 # Query
 @pytest.mark.parametrize(
-    "ip_with_dynamic_db, excepted", [("ip_with_postgreSQL", 3), ("ip_with_mySQL", 3)]
+    "ip_with_dynamic_db, excepted",
+    [
+        ("ip_with_postgreSQL", 3),
+        ("ip_with_mySQL", 3),
+        ("ip_with_mariaDB", 3),
+        ("ip_with_SQLite", 3),
+        ("ip_with_duckDB", 3),
+    ],
 )
 def test_query_count(ip_with_dynamic_db, excepted, request):
     ip_with_dynamic_db = request.getfixturevalue(ip_with_dynamic_db)
     out = ip_with_dynamic_db.run_line_magic("sql", "SELECT * FROM taxi LIMIT 3")
-    print("count out: ", len(out))
     assert len(out) == excepted
 
 
 # Create
 @pytest.mark.parametrize(
-    "ip_with_dynamic_db, excepted", [("ip_with_postgreSQL", 15), ("ip_with_mySQL", 15)]
+    "ip_with_dynamic_db, excepted",
+    [
+        ("ip_with_postgreSQL", 15),
+        ("ip_with_mySQL", 15),
+        ("ip_with_mariaDB", 15),
+        ("ip_with_SQLite", 15),
+        ("ip_with_duckDB", 15),
+    ],
 )
 def test_create_table_with_indexed_df(ip_with_dynamic_db, excepted, request):
     ip_with_dynamic_db = request.getfixturevalue(ip_with_dynamic_db)
     ip_with_dynamic_db.run_cell("results = %sql SELECT * FROM taxi LIMIT 15")
     ip_with_dynamic_db.run_cell("new_table_from_df = results.DataFrame()")
-    ip_with_dynamic_db.run_cell("%sql --persist sqlite:// new_table_from_df")
+    ip_with_dynamic_db.run_cell("%sql --persist new_table_from_df")
```

Comment:
yeah, let's check that calls to run_cell do not contain any exceptions with [error_in_exec](https://github.com/ipython/ipython/blob/main/IPython/core/interactiveshell.py#L3022)

Reply:
Fix in 
https://github.com/ploomber/jupysql/pull/117/commits/b2531c66499ce22ae13af393be13928255eb9918